### DISCUSSION TITLE
Versioning scheme update

### DISCUSSION THREAD

**Discussion by nocturneop15** [hassio-addons/addon-unifi#323] [Ideas]
Hello, 
I would really apreciate if versioning scheme could be updated so that version of this plugin will reflect version of unifi application - Unifi itself has a lot of version-specific glitches and consistent numbering scheme will be helpful for orientation.

**Comment by frenck** [comment]
Thanks for the suggestion, but that won't happen.
The add-on does not consist of just the Uni-Fi application, there are many more parts in here. For example, an OS, the communication layer with Home Assistant, and more of those.
